yql/plugins/weather.py

```
@@ -0,0 +1,35 @@
+''' Weather Plugins for YQL python wrapper  '''
+import yql
+
+class WeatherWrapper():
+    def __init__(self, FLICKR_API_KEY=None):
+        if FLICKR_API_KEY is not None:
+            self.FLICKR_API_KEY = FLICKR_API_KEY
+        else:
+            print("Flickr API Key required. Please signup for flickr api key")
```

Better to raise an exception here unless you choose to make the api_key required which will mean it will blow up automatically if the key isn't supplied.